source/Background/Controller.js

```diff
@@ -380,14 +377,20 @@ backgroundController.exposeController(
       storage.set({ apps: newApps });
     });
 
-    if (response) {
+    if (response?.accepted) {
       try {
         const publicKey = await keyring.getPublicKey();
+        // console.log('allow agent public key', publicKey, response);
+        // await plugProvider.createAgent(response.whitelist, response.host);
```

Comment:
Does this need to be uncommented?

Reply:
nice catch, not really, it should def be deleted